README.md

```diff
@@ -189,26 +184,29 @@ const { flags, values, positionals } = parseArgs({ args, options });
   - `"0o22"`
 - Does it coerce types?
   - no
-- Does `--no-foo` coerce to `--foo=false`?  For all flags?  Only boolean flags?
-  - no, it sets `{args:{'no-foo': true}}`
+- Does `--no-foo` coerce to `--foo=false`?  For all options?  Only boolean options?
+  - no, it sets `{values:{'no-foo': true}}`
```

Comment:
unrelated to this PR, but it would be nice to have parseArgs handle "what if i do `--foo --no-foo`" for me

Reply:
I note that's something you could trivially build on top of `parseArgs` if it reports indexes of arguments (https://github.com/pkgjs/parseargs/issues/84).